service/src/main/java/io/github/a13e300/tricky_store/KeystoreInterceptor.kt

```diff
@@ -65,23 +267,27 @@ object KeystoreInterceptor : BinderInterceptor() {
         reply: Parcel?,
         resultCode: Int
     ): Result {
-        if (target != keystore || code != getKeyEntryTransaction || reply == null) return Skip
+        if (target != keystore || code != getTransaction || reply == null) return Skip
         if (kotlin.runCatching { reply.readException() }.exceptionOrNull() != null) return Skip
         val p = Parcel.obtain()
         Logger.d("intercept post $target uid=$callingUid pid=$callingPid dataSz=${data.dataSize()} replySz=${reply.dataSize()}")
         try {
-            val response = reply.readTypedObject(KeyEntryResponse.CREATOR)
-            val chain = Utils.getCertificateChain(response)
-            if (chain != null) {
-                val newChain = CertHack.hackCertificateChain(chain)
-                Utils.putCertificateChain(response, newChain)
-                Logger.i("hacked cert of uid=$callingUid")
+            data.setDataPosition(104)
```

Comment:
What does this mean?

Reply:
L275 can change to `data.enforceInterface(DESCRIPTOR)`.
after `data.enforceInterface(DESCRIPTOR)`, dataPosition jump to 104

Reply:
I have pushed 1 commit clean code at L275 and remove useless code